### DISCUSSION TITLE
New RemuxExtensions

### DISCUSSION THREAD

**Discussion by C0nw0nk** [ptr727/PlexCleaner#141] [Q&A] [answered]
So i have been using this its been working well.

I did see remux extensions could use some more files that Plex does not support. Convert them all to MKV for direct playback compatibility.

```
"ReMuxExtensions": [
".avi",
".mxf",
".m2p",
".ps",
".m2ts",
".mts",
".ts",
".tsv",
".vob",
".evo",
".3gp"
".3g2".
".f4v",
".flv",
".mp4",
".m4v",
".asf",
".wmv",
".dv",
".mov",
".qt",
".ogg",
".ogv",
".ogx",
".webm",
".rmvb",
".divx",
],
```

There are loads more https://en.wikipedia.org/wiki/Comparison_of_video_container_formats

**Comment by ptr727** [comment, marked as answer]
You are welcome to add as many as you want, the default values are based on my personal preference.
